src/Geolocator.Plugin.WindowsPhone81/GeolocatorImplementation.cs

```diff
@@ -172,6 +172,26 @@ public Task<Position> GetPositionAsync(TimeSpan? timeout, CancellationToken? can
         }
 
         /// <summary>
+        /// Gets position async and reverse geocode
+        /// </summary>
+        /// <returns>Address of the current position</returns>
+        public Task<Address> ReverseGeocodeCurrentLocation()
+        {
+            throw new NotImplementedException();
```

Comment:
Same as above, I assume there has to be an implementation somewhere in Xamarin.Forms world?